examples/json/FaxSendRequestExample.json

```
@@ -0,0 +1,12 @@
+{
+  "file": "<form-data file upload>",
+  "file_url[0]": "<link to file url>",
```

Json does not use formdata notation `[0]` for arrays:

```suggestion
  "file_urls": [
    "https://www.dropbox.com/s/ad9qnhbrjjn64tu/mutual-NDA-example.pdf?dl=1"
  ],
```